lib/jellyfish/component/hls.ex

```diff
@@ -20,34 +22,37 @@ defmodule Jellyfish.Component.HLS do
 
   @impl true
   def config(options) do
-    storage = fn directory -> %LLStorage{directory: directory, room_id: options.room_id} end
-    RequestHandler.start(options.room_id)
-
-    {:ok,
-     %HLS{
-       rtc_engine: options.engine_pid,
-       owner: self(),
-       output_directory: output_dir(options.room_id),
-       mixer_config: %MixerConfig{
-         video: %CompositorConfig{
-           stream_format: %Membrane.RawVideo{
-             width: 1920,
-             height: 1080,
-             pixel_format: :I420,
-             framerate: {24, 1},
-             aligned: true
+    with {:ok, valid_opts} <- OpenApiSpex.cast_value(options, ApiSpec.Component.HLS.schema()) do
```

Comment:
That's because RTSP is further working on the options passed to `cast_value`, meanwhile we're only using one flag here in HLS. Same thing happens in WebRTC with `enableSimulcast` flag